var/spack/repos/builtin/packages/cool/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cool(CMakePackage):
+    """COOL provides specific software components and tools for the handling of the time
+       variation and versioning of the experiment conditions data."""
+
+    homepage = "https://coral-cool.docs.cern.ch/"
+    git      = "https://gitlab.cern.ch/lcgcool/cool.git"
+
+    tags = ['hep']
```

Comment:
@iarspider Would you sign up for being a maintainer? :grimacing:

Reply:
Yes.